addons/payment/security/ir.model.access.csv

```diff
@@ -10,4 +10,6 @@ payment_method_portal,payment.token.portal,model_payment_token,base.group_portal
 payment_method_system,payment.token.system,model_payment_token,base.group_system,1,1,1,1
 payment_icon_all,payment.icon.all,model_payment_icon,,1,0,0,0
 payment_icon_user,payment.icon.user,model_payment_icon,base.group_user,1,1,1,0
-payment_icon_system,payment.icon.system,model_payment_icon,base.group_system,1,1,1,1
\ No newline at end of file
+payment_icon_system,payment.icon.system,model_payment_icon,base.group_system,1,1,1,1
+"access_payment_acquirer_onboarding_wizard","access.payment.acquirer.onboarding.wizard","model_payment_acquirer_onboarding_wizard","base.group_system",1,0,1,0
+"access_payment_link_wizard","access.payment.link.wizard","model_payment_link_wizard","account.group_account_user",1,0,1,0
```

Comment:
this last one should also be accessible for `sales_team.group_sale_salesman`; could you add an equivalent ACL in the `sale` module for that group (`sale` depends on `payment`)